workflows/rnaseq/main.nf

```diff
@@ -142,7 +142,7 @@ workflow RNASEQ {
     .reads
     .mix(ch_fastq.single)
     .set { ch_cat_fastq }
-    ch_versions = ch_versions.mix(CAT_FASTQ.out.versions.first().ifEmpty(null))
```

Comment:
@pinin4fjords is this required?

Reply:
There's something odd going on there- CAT_FASTQ etc moved to the subworkflow 'fastq_qc_trim_filter_setstrandedness' some time ago- and we removed all the `ifEmpty()`, since they were causing problems.

Reply:
Think I just need to resolve the conflicts and this will vanish.